.github/pull_request_template.md

```diff
@@ -6,10 +6,9 @@
  
 ### Check List
 - [ ] New functionality includes testing.
-  - [ ] All tests pass
 - [ ] New functionality has been documented.
   - [ ] New functionality has javadoc added
```

Comment:
Should this be added considering there are build files, config files which would not have javadocs?

Reply:
This should remain as is. There are a few types of documentation within our project (README, docs/, javadocs, etc.). Given we are publishing an API, I think it is important to explicitly call this out.